README.md

```diff
@@ -12,22 +12,28 @@
+[Docs](https://data-verse.gitbook.io/docs/) • [Examples](https://github.com/UpstageAI/dataverse/tree/main/examples) • [API Reference](https://data-verse.readthedocs.io/en/latest/) • [FAQ](https://data-verse.gitbook.io/docs/documents/faqs) • [Contribution Guide](https://github.com/UpstageAI/dataverse/blob/main/contribution/CONTRIBUTING.md)  • [Contact](mailto:dataverse@upstage.ai)  • [Discord](https://discord.gg/aAqF7pyq4h)
 <br><br>
 <div align="left">
 
 ## Welcome to Dataverse!
 Dataverse is a freely-accessible open-source project that supports your **ETL(Extract, Transform and Load) pipeline with Python**. We offer a simple, standardized and user-friendly solution for data processing and management, catering to the needs of data scientists, analysts, and developers in LLM era. Even though you don't know much about Spark, you can use it easily via _dataverse_.
 
-### Why should I use Dataverse?
-- **Integrated library**: Dataverse streamline your workflow by integrating multiple preprocessing libraries into one, eliminating the hassle of settings and searching for the right tools. You can even use HuggingFace datasets from hub directly into the pipeline.
-- **Simplified Spark usage**: You don't have to be a pro with Spark. With just a few setting configurations, you can easily take Spark's high performance effortlessly.
-- **Facilitated collaboration**: Offer uniform preprocessing codes to ensure consistent results whether who runs the code. Dataverse also enable collaboration among users with varying levels of Spark proficiency.
+### With Dataverse, you are empowered to
+
+- utilize a range of preprocessing functions without the need to install multiple libraries.
+- create high-quality data for analysis and training of Large Language Models (LLM).
+- leverage Spark with ease, regardless of your expertise level.
+- facilitate smoother collaboration among users with varying degress of Spark proficiency.
+- enjoy freedom from the limitations of local environments by harnessing the capabilities of AWS EMR.
 
 ### Key Features of Dataverse
-- **Block-Based:** Dataverse lets you build Spark code like putting together puzzle pieces. You can easily add, take away, or rearrange pieces to get the results you want.
-- **Configure-Based:** Dataverse has a user-friendly setup where you don't need to know all the code. Just set up the options, and you're good to go.
-- **Extensible:** It's designed to meet your specific demands, allowing for custom features that fit perfectly with your project.
+- **Block-Based**: In Dataverse, a `block` means a `ETL registered function` which is running on Spark. You can build Spark code like putting together puzzle pieces. You can easily add, take away, or re-arrange pieces to get the results you want via configure.
```

Comment:
How about changing the term "ETL registered function" to "registered ETL function"?